gguf-py/scripts/gguf-template.py

```diff
@@ -0,0 +1,140 @@
+"""
+gguf-template.py - example file to extract the chat template from the models metadata
+"""
+
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import sys
+from pathlib import Path
+
+import jinja2
+
```

Comment:
Not very critical in a script like this, but nevertheless, add:
```python
from jinja2.sandbox import ImmutableSandboxedEnvironment
```

Reply:
I just saw and read the CVE. Will patch.